kyuubi-common/src/main/scala/org/apache/kyuubi/util/ThreadUtils.scala

```diff
@@ -109,4 +109,28 @@ object ThreadUtils extends Logging {
     thread.setUncaughtExceptionHandler(NamedThreadFactory.kyuubiUncaughtExceptionHandler)
     thread.start()
   }
+
+  /**
+   * Schedule a runnable to the scheduled executor service.
+   * The exceptions thrown in the runnable will be caught and logged.
+   */
+  def scheduleTolerableRunnableWithFixedDelay(
+      scheduler: ScheduledExecutorService,
+      runnable: Runnable,
+      initialDelay: Long,
+      delay: Long,
+      timeUnit: TimeUnit,
+      errorMessageHint: String): Unit = {
+    scheduler.scheduleWithFixedDelay(
+      () =>
+        try {
+          runnable.run()
+        } catch {
+          case e: Exception =>
+            error(s"Error in $errorMessageHint", e)
```

Comment:
```
case e: Throwable => error(s"Uncaught exception in thread ${Thread.currentThread().getName}", t)
```

Reply:
You can remove `errorMessageHint` variable.

It is ok to use the thread name as hint.

Reply:
```
case e: Throwable => error(s"Uncaught exception in thread ${Thread.currentThread().getName}", t)
```

Reply:
Sure, it's much more elegant.